apps/src/code-studio/initApp/project.js

```diff
@@ -559,7 +559,15 @@ var projects = module.exports = {
         return null;
     }
   },
-  /**
+
+  canServerSideRemix() {
+    // The excluded app types need to make modifications to the project that
+    // apply to the remixed project, but should not be saved on the original
+    // project. See (Turtle|Studio).prepareForRemix().
+    return !['turtle', 'studio'].includes(appOptions.app);
```

Comment:
Ah, yeah it used to force you to log in and I broke that. Fixed by just disabling the last save.